dashboard/app/models/script.rb

```diff
@@ -169,6 +169,10 @@ def has_lesson_plan?
     k5_course? || %w(msm algebra).include?(self.name)
   end
 
+  def show_freeplay_links
```

Comment:
I think it would follow the existing pattern better if this was named `show_freeplay_links?`. otherwise, lgtm